### PR TITLE
add a remote room file option

### PR DESCRIPTION
**Pull request by juliemar**
### Description
We have some issues talking about difficult to manage the environment variable `ROOMS_DATA` 
[Error when using custom ROOMs (docker compose env)](https://github.com/ResultadosDigitais/matrix/issues/297)
[Matrix not reading ROOMS_DATA from env file](https://github.com/ResultadosDigitais/matrix/issues/270)
[ROOMS single line environment variable is tuff to use when you have too many rooms](https://github.com/ResultadosDigitais/matrix/issues/269)

This PR I put a new option to read a remote `.json` file.  

### How to test?
Setup Environment variable `ROOMS_SOURCE=REMOTE` and the `ROOMS_DATA` with a link to the `json` room's source file.

### Expected behavior
#matrix needs to make an office with the virtual rooms configured in the remote `.json` file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +109,10 @@ The **#matrix** project has some environment variables that important to define.
 		   }
 		 ]
 
+Other option is to have a remote rooms config file. You can to configure a `ROOMS_SOURCE=REMOTE` and config `ROOMS_DATA` like the example:
```
Comment (Verhaeg) [inline README.md:112]: Little typo here:

> Another option is to have a remote rooms config file. You can configure `ROOMS_SOURCE=REMOTE` and config `ROOMS_DATA` like the example:

Maybe also add a note that the file must be publicly accessible in this case.

Comment (juliemar) [inline README.md:112]:
```suggestion
Another option is to have a remote rooms config file (this file needs to be accessible via http/s). You can configure a `ROOMS_SOURCE=REMOTE` and config `ROOMS_DATA` like the example:
```

Comment (juliemar) [reply README.md:112]: @Verhaeg what do you think?

Comment (Verhaeg) [reply README.md:112]: Great! 
and awesome addition to the repository!

File: backend/app/controllers/rooms.controller.js [hunk 2]
```diff
@@ -55,11 +56,31 @@ const fetchFromEnvironment = (env) => {
...
+            } catch(e) {
+                reject(e);
+            }
+        });
+    });
+};
+
 const fetchRooms = (strategy) => {
   switch (strategy) {
     // TODO add suport to fetch from endpoint
```
Comment (angeliski) [inline backend/app/controllers/rooms.controller.js:79]:
```suggestion
```

**Comment by juliemar** [comment]
> A suggestion, we can verify in the app.healthcheck.js if the rooms are loaded correctly
I can do that in another PR if you prefer.

Good Idea @angeliski I don't have an idea how to do it :) I'll merge this PR and you can open another with this check :rocket:

**Pull request merged**